microsoft-edge/devtools-guide-chromium/accessibility/emulate-vision-deficiencies.md

```diff
@@ -86,5 +86,6 @@ You may also use **Command Menu** to access the different simulations.
 [DevtoolsRenderingToolsIndex]: ../rendering-tools/index.md "Analyze runtime performance | Microsoft Docs"  
 
 [ColorblindawarenessMain]: http://www.colourblindawareness.org "The Colour Blind Awareness organisation"  
+<!-- this link must be http, not https -->  
```

Comment:
This link works: https://www.colourblindawareness.org/ and is https so I would just add it here